### PR TITLE
Myer/add python 3 tests #5

### PR DESCRIPTION
**Pull request by codekiln**

### CODE REVIEW

File: django_elastic_migrations/urls.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function, unicode_literals)
```
Comment (gnulnx) [inline django_elastic_migrations/urls.py]: This line isn't necessary for python3 any longer....though it does not harm at all to leave it.

**Comment by gnulnx** [comment]
@codekiln   You should check out . https://coveralls.io/

I find it to be a really solid code coverage tool that link in seemlessly with travis

File: django_elastic_migrations/urls.py (hunk 1 above)
Comment (codekiln) [reply django_elastic_migrations/urls.py]: Until we're ported 100%, I was planning to support both 2 and 3, so I'll leave it in for py2's sake for now.

**Comment by codekiln** [comment]
Tests in progress @ https://hbx-ci.myhbx.org/api/1443/

File: django_elastic_migrations/utils/es_utils.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 def get_index_hash_and_json(index):
     spec = index.to_dict()
-    json_str = json.dumps(spec, sort_keys=True)
+    json_str = json.dumps(spec, sort_keys=True).encode('utf-8')
```
[the remark this answers is not in the document]
Comment (codekiln) [reply django_elastic_migrations/utils/es_utils.py]: Great question. Yes, it's just necessary for py3, and works fine with py2.  See https://github.com/HBS-HBX/django-elastic-migrations/runs/10652762, which is the build checks for py3 for commit 3044d2e, which doesn't include the encoding declaration. You can see the py2 tests passed, and the py3 test run failed with "TypeError: Unicode-objects must be encoded before hashing"

**Pull request merged**